package.json

```diff
@@ -13,7 +13,7 @@
 		"format",
 		"metalua"
 	],
-	"publisher": "Alex",
+	"publisher": "arivero",
```

Comment:
Not that we don't love our users and want them to have the best experience ever .... but with your info here, it's possible that you could be contacted forever and ever for feature updates and requests and problems and etc etc etc ... Just a thought ... ;->

Also, would you please also set tabs to just two spaces for JSON files (as well as JS files). :-)

Reply:
Yeah I know that could cause some work later on but the problem is that I need to make myself a publisher in order to publish the extension into the marketplace.

Reply:
Ah, the marketplace. Cool!!!